app/admin/friendship.rb

```diff
@@ -0,0 +1,37 @@
+ActiveAdmin.register Approval, as: "Friendship" do
+  actions :all, except: [:new, :create]
+
+  controller do
+    def scoped_collection
+      end_of_association_chain.where id: Approval.find_by_sql(
+        <<-SQL
+          select distinct on (token) *
```

Comment:
though one-by-one is not necessary, I think

@t-harps could you please look at CI error related to load.sumome.com script?